src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/employee_name_with_avatar.html.twig

```diff
@@ -24,13 +24,10 @@
  *#}
 
 {% if record.employee is not null %}
-  {%
-    set employeeName, employeeImage = record.employee, 'https://profile.prestashop.com/'~record.email|url_encode~'.jpg'
-  %}
   <span class="employee_avatar_small">
-    <img class="img rounded-circle" alt="{{ employeeName }}" src="{{ employeeImage }}" height="32" width="32" />
+    <img class="img rounded-circle" src="{{ asset(record.image) }}" height="32" width="32" />
```

Comment:
Why remove the `alt` ? Opquast will not be happy 😅

Reply:
Please check the issue. It has been asked by @eternoendless because when the image doesn't load, it displays twice the employee name.

Reply:
Ha ! I see

Then I think the real issue is rather than the `alt` content is duplicate content with something else in the page.

Maybe the `alt` content should be then `{ {record.employeeName}} avatar` instead. But I'm pretty sure the ALT is a good thing. If I remember correctly Opquast training, the alt description must help people who cant see the image understand what they are missing.

Reply:
The alt is for screen readers, so when the screen reader reads the page : it reads "Employee Name Employee Name"

Reply:
Please see with PM for decision.

Reply:
@eternoendless Franck tells me you are the one who asked to remove the ALT tag.
I am very surprised 😮 this is the 1st time I see a usecase where it would be a good thing to NOT have a ALT tag.

I understand the duplicate content between the alt and the row content is a bad thing.

May I suggest to _improve_ the ALT content ? for example
- "employee Mathieu Ferment avatar"
- "employee Mathieu Ferment picture"
-  "employee image"

This way the ALT does its job (= provide a textual alternative for people unable to see the image) AND we avoid the duplicate content.

Reply:
🆙